pkg/scheduler/framework/plugins/noderesources/fit.go

```diff
@@ -88,6 +88,7 @@ type Fit struct {
 	ignoredResourceGroups           sets.Set[string]
 	enableInPlacePodVerticalScaling bool
 	enableSidecarContainers         bool
+	enableQHint                     bool
```

Comment:
Could you use the same field name for all plugins, e.g. `qHintEnabled`?

Reply:
Unified with the name `enableSchedulingQueueHint`. (because some already have the field named `enableSchedulingQueueHint`.